libs/lib/utils.cpp

```diff
@@ -270,17 +270,17 @@ Utils::ShiftTrace(fftw_real * trace,
 
   std::vector<fftw_real> l_trace(fft_large,0);
   fftw_complex * l_trace_c = reinterpret_cast<fftw_complex*>(&(l_trace[0]));
-  for(size_t i=0;i<fft_small;i++)
+  for(i=0;i<fft_small;i++)
```

Comment:
De fleste endringene er av denne typen. Det var definert {{size_t i;}} utenfor, og da ble det warning når man bruker {{for(size_t i=0...)}}